error[E0308]: mismatched types in argminmax-0.6.2

### Checks

- [x] I have checked that this issue has not already been reported.
- [x] I have confirmed this bug exists on the [latest version](https://pypi.org/project/polars/) of Polars.

### Reproducible example

build failure

### Log output

```shell
error[E0308]: mismatched types
     --> /usr/ports/misc/py-polars/work-py311/polars-py-1.29.0/py-polars/cargo-crates/argminmax-0.6.2/src/simd/simd_i32.rs:195:32
      |
195   |             _mm512_loadu_si512(data as *const i32)
      |             ------------------ ^^^^^^^^^^^^^^^^^^ expected `*const __m512i`, found `*const i32`
      |             |
      |             arguments to this function are incorrect
      |
      = note: expected raw pointer `*const std::arch::x86_64::__m512i`
                 found raw pointer `*const i32`
note: function defined here
     --> /usr/local/lib/rustlib/src/rust/library/core/src/../../stdarch/crates/core_arch/src/x86/avx512f.rs:34452:15
      |
34452 | pub unsafe fn _mm512_loadu_si512(mem_addr: *const __m512i) -> __m512i {
      |               ^^^^^^^^^^^^^^^^^^

For more information about this error, try `rustc --explain E0308`.
error: could not compile `argminmax` (lib) due to 2 previous errors
warning: build failed, waiting for other jobs to finish...
💥 maturin failed
  Caused by: Failed to build a native library through cargo
  Caused by: Cargo build finished with "exit status: 101": `env -u CARGO PYO3_ENVIRONMENT_SIGNATURE="cpython-3.11-64bit" PYO3_PYTHON="/usr/local/bin/python3.11" PYTHON_SYS_EXECUTABLE="/usr/local/bin/python3.11" "cargo" "rustc" "--message-format" "json-render-diagnostics" "--manifest-path" "/usr/ports/misc/py-polars/work-py311/polars-py-1.29.0/py-polars/Cargo.toml" "--release" "--lib"`
Error: command ['maturin-3.11', 'pep517', 'build-wheel', '-i', '/usr/local/bin/python3.11', '--compatibility', 'off'] returned non-zero exit status 1
```

### Issue description

See the log.

### Expected behavior

n/a

### Installed versions

Building version 1.29.0

rust-nightly-1.88.0.20250427
py311-maturin-1.8.3

--- a/crates/polars-ops/Cargo.toml
+++ b/crates/polars-ops/Cargo.toml
@@ -17,7 +17,7 @@ polars-schema = { workspace = true }
 polars-utils = { workspace = true }
 
 aho-corasick = { workspace = true, optional = true }
-argminmax = { version = "0.6.2", default-features = false, features = ["float"] }
+argminmax = { version = "0.6.3", default-features = false, features = ["float"] }
 arrow = { workspace = true }
 base64 = { workspace = true, optional = true }
 bytemuck = { workspace = true }

